src/screens/SpeedUpAndCancelSheet.js

```diff
@@ -293,6 +293,10 @@ export default function SpeedUpAndCancelSheet() {
           setMinGasPrice(calcMinGasPriceAllowed(hexGasPrice));
         } catch (e) {
           logger.log('something went wrong while fetching tx info ', e);
+          logger.sentry(
+            'Error speeding up or canceling transaction: [transaction]',
+            tx
+          );
           captureException(e);
```

Comment:
can you create a custom error like

` const customError = new Error('Error SpeedupOrCancelSheet');`

and capture that exception instead? that way we know exactly what to look for.

Reply:
Np. I captured the custom exception and added another `logger.sentry` call to make sure we still capture the actual exception in case there are multiple. Let me know if that makes sense. Still figuring out sentry.